Maple2Storage/Types/Metadata/ItemMetadata.cs

```diff
@@ -132,7 +132,7 @@ public class ItemSkillMetadata
     [XmlElement(Order = 1)]
     public int SkillId;
     [XmlElement(Order = 2)]
-    public int SkillLevel;
+    public short SkillLevel;
```

Comment:
if skill level is short, can we change it to short everywhere else?

Reply:
Yeah I think that'd be ideal. Do you know of any places where it is int right now?

Reply:
looks like you got it in SkillMetadata